2024/calcite-react/demo/public/assets/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
ha, I didn't know you could gitignore a gitignore using `*` 😁

Reply:
(optional) do you want to instead gitignore the entire assets directory by creating a `.gitignore` like this in the 2024/calcite-react/demo/ directory directly?

```yaml
/public/assets
```